src/bug-filing/github/create-github-bug-filing-url.ts

```diff
@@ -53,6 +54,7 @@ export function createGitHubBugFilingUrl(
 
     const title = buildTitle(bugData);
     const body = buildGithubText(bugData);
+    const repository = IssueFilingUrlStringUtils.appendSuffixToUrl(settingsData.repository, 'issues');
```

Comment:
Should way for the `IssueFilingUrlStringUtils` refactor before completing this PR. We should not use objects as globals, this should be pass as a dependency.

Reply:
Sure.